recipes/treelite/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "treelite" %}
+{% set version = "0.32" %}
+{% set sha256 = "ac34d0f925dad06d1c90932a57e805e6f0361988f47a457936997fb3aad17cfa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/dmlc/treelite
+  git_rev: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<17]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - scipy
+  run:
+    - python
+    - scipy
+
+test:
+  imports:
+    - treelite
```

Comment:
I think there are some sub-packages that should probably imported in the tests

Reply:
You mean shared/static libs?
The package compiles c++ code with a c api and then uses ctypes inside python.